torchgeo/datamodules/so2sat.py

```diff
@@ -58,29 +58,27 @@ class So2SatDataModule(pl.LightningDataModule):
 
     def __init__(
         self,
-        root_dir: str,
         batch_size: int = 64,
         num_workers: int = 0,
-        bands: str = "rgb",
+        band_set: str = "rgb",
```

Comment:
Renamed from `bands` to `band_set` to match `SEN12MSDataModule` and to remove ambiguity between `So2SatDataModule` and `So2Sat` where `bands` is a list of band names.